views/folders.templ

```diff
@@ -9,21 +9,15 @@ import (
 	"github.com/ugent-library/deliver/models"
 )
 
-templ Folders(c *ctx.Ctx, folders []*models.Folder, totalFolders int) {
+templ Folders(c *ctx.Ctx, space *models.Space, folders []*models.Folder, pagination *models.Pagination) {
 	<div class="card w-100 mb-6">
 		<div class="card-header">
-			<div class="bc-toolbar">
-				<div class="bc-toolbar-left">
-					<div class="bc-toolbar-item">
-						<h2>Folders</h2>
-					</div>
-				</div>
-				<div class="bc-toolbar-right">
-					<div class="bc-toolbar-item">
-						<p>Showing { fmt.Sprint(len(folders)) } of { fmt.Sprint(totalFolders) } folders</p>
-					</div>
-				</div>
-			</div>
+			@Pagination(PaginationArgs{
+				pagination:     pagination,
+				baseHref:       c.Path("space", "spaceName", space.Name),
+				baseHtmxGetUrl: c.Path("getFolders", "spaceName", space.Name),
```

Comment:
Not sure I like it too much that we need to pass on two base paths. One for full reload `a[href]` and one used for the `hx-get` attribute. This may be possible with just the `href` and using `hx-boost` with a target instead but then the "space" handler should consider the `HX-Request` header to decide which response to return.

Reply:
no that's partially why I haven't merged it yet, it all seems a bit clunky.
biblio-backoffice also has a pagination component with pager ellipsis, maybe we can split it off ans use it as a common component.

Reply:
But that one is still gohtml without HTMX so that wouldn't be possible to integrate here, right?

I've also looked at the [ellipsis calculation of that Pagination model](https://github.com/ugent-library/biblio-backoffice/blob/main/pagination/pagination.go#L78-L194) but it was almost 120 lines of undocumented juggling with slices that I could not understand so wouldn't want to maintain that.